packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -304,17 +308,17 @@ interface TypingDataFromIndividualTypeScriptVersion {
 
 /**
  * @param typescriptVersion Set if this is in e.g. a `ts3.1` directory.
- * @param packageName Name of the outermost directory; e.g. for "node/v4" this is just "node".
+ * @param typesDirectoryName Name of the outermost directory; e.g. for "node/v4" this is just "node".
  * @param ls All file/directory names in `directory`.
  * @param fs FS rooted at the directory for this particular TS version, e.g. `types/abs/ts3.1` or `types/abs` when typescriptVersion is undefined.
  */
-function getTypingDataForSingleTypesVersion(
+function getFilesForSingleTypeScriptVersion(
   typescriptVersion: TypeScriptVersion | undefined,
-  packageName: string,
+  typesDirectoryName: string,
```

Comment:
I wonder if we should have two different string types for packages and typeDirectories...